model/validate.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * Validating onboarding form
+ * @author Laxmi kandel
+ */
+class Validate
+{
+    /**
+     * @var array
+     */
+    private $_errors;
+
+    /**
+     * Validator constructor.
+     */
+    public function __construct()
+    {
+        $this->_errors = array();
+    }
+
+    /**
+     * @return array
+     */
+    public function getErrors()
+    {
+        return $this->_errors;
+    }
+
+    /**
+     * Validating First Name
+     * @param $firstName tutor's first name
+     * @return bool
```

Comment:
You use @return bool several times in this file and all of them need a description of what is being returned. Tell readers what it means when the function returns true